sdk/keyvault/keyvault-certificates/review/keyvault-certificates.api.md

```diff
@@ -390,16 +337,15 @@ export interface OrganizationDetails {
 export { PipelineOptions }
 
 // @public
-export interface RecoverDeletedCertificateOptions extends coreHttp.OperationOptions {
+export interface PurgeDeletedCertificateOptions extends coreHttp.OperationOptions {
```

Comment:
These two renames seems a little odd

Reply:
ohh checking what happened here

Reply:
Both PurgeDeletedCertificateOptions and RecoverDeletedCertificateOptions are in the review file! I believe they just got moved. Please let me know if I'm missing something!